asciidocs/setup.adoc

```diff
@@ -124,49 +124,47 @@ This should come up with the Nextflow version and runtime information:
+- Gitpod gives you 50 hours of free environment runtime per month.
+- It includes up to 16 CPUs and 30GB of workspace.
+- Gitpod will time out after 30 minutes. However, changes are saved for up to 2 weeks (see the next section for reopening a timed out session).
 
-See www.gitpod.io for more details.
+See https://www.gitpod.io[gitpod.io] for more details.
 
 === Reopening a Gitpod session
 
-You can reopen an environment by going to https://gitpod.io/workspaces and finding your previous environment, then clicking the button with three dots and selecting Open.
+You can reopen an environment from https://gitpod.io/workspaces. Find your previous environment in the list, then select the ellipsis (three dots icon) and select Open.
 
-If you save the URL from your previous Gitpod environment, you can just paste this into your browser to open the previous environment.
+If you have saved the URL for your previous Gitpod environment, you can simply open it your browser to open the previous environment.
 
 Alternatively, you can start a new workspace by following the Gitpod URL:
 https://gitpod.io/#https://github.com/seqeralabs/nf-training-public
 
-This tutorial provides all the scripts, so don't worry if you have lost your environment. In the `nf-training` directory, you can find the main scripts used in the tutorial.
+If you have lost your environment, you can find the main scripts used in this tutorial in the `nf-training` directory to resume with a new environment. 
 
 === Saving files from Gitpod to your local machine.
 
-To save your files, select your file of interest from the explorer panel, then right click the file to click `Download`
+To save any file from the explorer panel, right-click the file and select `Download`.
 
 === Training material
 
-If you close the material by accident, the following URL will open the training course in your browser:
-
-https://training.seqera.io/
+The training course can be accessed in your browser from https://training.seqera.io/.
 
 == Selecting a Nextflow version
 
 By default, Nextflow will pull the latest stable version into your environment.
 
 However, Nextflow is constantly evolving as we make improvements and fix bugs.
 
-It is worth checking out the latest releases on github https://github.com/nextflow-io/nextflow[here].
+The latest releases can be viewed on GitHub https://github.com/nextflow-io/nextflow[here].
 
-If you want or need to use a specific version of Nextflow, you can set the NXF_VER variable as shown below:
+If you want to use a specific version of Nextflow, you can set the NXF_VER variable as shown below:
 
 [source,bash,linenums]
 ----
 export NXF_VER=21.10.0
 ----
 
-WARNING: If you have changed the version to `21.10.0`, as above, most of this tutorial workshop will not work, as we need NXL_VER=22 or later to use DSL2 as default.
+WARNING: Most of this tutorial workshop requires NXF_VER=22 or later, to use DSL2 as default.
```

Comment:
The versioning scheme relies on `yy.mm.patch` scheme. https://github.com/nextflow-io/nextflow/tags 

```suggestion
WARNING: Most of this tutorial workshop requires NXF_VER=22.04.5 or later, to use DSL2 as default.
```